docs/changelog.rst

```diff
@@ -13,7 +13,21 @@ Added
 - Experiments:
     - Script to simulate the critical temperature of SiQAD and Bestagon gates
 - Algorithms:
-    - Added support for different ways of implementing input information in SiDB technology to the BDL input iterator
+    - Physical Design:
+        - Added an option to specify discretionary cost objectives to GOLD
+        - Added a flag to enable multi-threading to GOLD
+        - Added a timeout option to post-layout optimization
+    - Simulation:
```

Comment:
I think, *QuickCell* belongs more to physical design. Maybe we can put it there. Thank you! 🙏